spec/components/notice_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+describe "Notice", type: :view do
+  def component_name
+    "notice"
+  end
+
+  it "fails to render when no parameters are given" do
```

Comment:
I'd change the name of this test to something like "renders nothing when no params provided". The current test name suggests we should get an error or failure.